images/node/example/Dockerfile

```diff
@@ -1,7 +1,10 @@
 # syntax=docker/dockerfile:1
 # Based on https://docs.docker.com/language/nodejs/build-images/
 
-FROM cgr.dev/chainguard/node
+ARG BASE=cgr.dev/chainguard/node
+
+FROM ${BASE}
+
```

Comment:
We had not been effectively testing the just-built image, and instead only tested the already-tagged-`:latest` image on every change. This is fixed now!